turbinia/client_test.py

```diff
@@ -359,6 +359,39 @@ def testClientFormatTaskStatusFiles(self, _, __, ___):
         'inst', 'proj', 'reg', all_fields=True, full_report=True)
     self.assertEqual(result.strip(), LONG_REPORT_FILES.strip())
 
+  @mock.patch('libcloudforensics.providers.gcp.internal.function.GoogleCloudFunction.ExecuteFunction')  # yapf: disable
+  @mock.patch('turbinia.client.task_manager.PSQTaskManager._backend_setup')
+  @mock.patch('turbinia.state_manager.get_state_manager')
+  def testClientFormatRequestStatus(self, _, __, ___):
+    """Tests format_request_status() with default days."""
+    client = TurbiniaClientProvider.get_turbinia_client()
+    client.get_task_data = mock.MagicMock()
+    client.get_task_data.return_value = self.task_data
+    result = client.format_request_status('inst', 'proj', 'reg')
+    self.assertIn('Requests made within 7 days', result.strip())
```

Comment:
Rather than looking for one string in the report here, could we potentially put the full output in a variable at near the top of the file and compare the full test output (similar to the test above this one does with `LONG_REPORT_FILES`).  That would help catch issues like the one with the enumerated set above.  It's also kind of nice because you have an example of what the full expected output is. 

(Hint: You might already use this trick, but to get the report output from the test data you can just do an `assertEqual(REPORT, '')` against an empty string, and the failure output will print out what it's comparing with, and then you can just copy/paste that into the variable).

Reply:
Thanks for the hint, was pretty helpful. Added this test in!